### PR TITLE
HeatMap integration

### PR DESCRIPTION
**Pull request by anjaneyulukinnara**

### CODE REVIEW

**Comment by anjaneyulukinnara** [comment]
Added heatmap @theTechie pls review

File: go-corona-app/screens/HomeScreen.js [hunk 1]
```diff
@@ -1,138 +1,121 @@
...
 import { ScrollView } from 'react-native-gesture-handler';
+import MapView, { PROVIDER_GOOGLE, Heatmap } from 'react-native-maps';
+import Constants from 'expo-constants';
+import * as Location from 'expo-location';
+
+import data from '../test_data/location_history.js'
+
 
 import { MonoText } from '../components/StyledText';
+import { renderer } from 'react-test-renderer';
```
Comment (theTechie) [inline go-corona-app/screens/HomeScreen.js:13]: @anjaneyulukinnara - any idea why we need a test renderer?

**Review by theTechie — APPROVED** [review]
Works with dummy data. We can spot headmaps near Warangal area

File: go-corona-app/screens/HomeScreen.js (hunk 1 above)
Comment (anjaneyulukinnara) [reply go-corona-app/screens/HomeScreen.js:13]: no need, I used to write in typescript which shows unused imports, not the case here will remove in next pull

**Pull request merged**